packages/core/pluggableElementTypes/models/baseTrackConfig.ts

```diff
@@ -38,6 +38,11 @@ export function createBaseTrackConfig(pluginManager: PluginManager) {
           'list of which feature attributes to index for text searching',
         defaultValue: ['Name', 'ID', 'Description'],
       },
+      indexingFeatureTypesToExclude: {
+        type: 'stringArray',
+        description: 'list of feature types to exclude in text search index',
+        defaultValue: ['CDS', 'exon', 'transcript'],
+      },
```

Comment:
why don't we call this `textSearchIndexingFeatureTypesToExclude` or something more descriptive

Reply:
modified